lib/tag-handler.js

```diff
@@ -0,0 +1,56 @@
+const { log, exec } = require('./utils')
+const { githubHandler } = require('./github')
+const packageJson = require('../package.json')
+
+
+module.exports.tagOptions = (yargs) => yargs
+  .option('github', {
+    type: 'boolean',
+    alias: 'gh',
+    description: 'Flag to create a tag from commit version and execute a GitHub release',
+    default: false,
+  }).option('number', {
+    type: 'number',
+    alias: 'n',
+    description: 'Limit the number of commits to read tag from',
+    default: 3,
+  }).option('verbose', {
+    type: 'boolean',
+    alias: 'v',
+    describe: 'Show verbose output',
+    default: false,
+  })
+
+const versionPattern = /([0-9]\d*\.[0-9]\d*\.[0-9]\d*)(-alpha||-beta)?(-[0-9]\d*)?/
```

Comment:
should we open up `alpha|beta` portion to be more flexible to cover cases such as `rc` (release candidate) and other perculiar (but not impossible) practices?